decisions/wc_offmap_decisions.txt

```diff
@@ -580,7 +580,7 @@ offmap_decisions = {
 					lacks_offmap_power_trigger = yes
 				}
 			}
-			FROM = { NOT = { has_character_flag = requested_offmap_power } }
+			hidden_effect = { FROM = { NOT = { has_character_flag = requested_offmap_power } } }
```

Comment:
hidden_effect don't work in conditions

<details>

![image](https://user-images.githubusercontent.com/46576860/63262833-ad975a00-c28f-11e9-8821-30566f0e5e9d.png)

</details>

Reply:
fixed